praw/models/reddit/mixins/__init__.py

```diff
@@ -139,6 +139,52 @@ def remove(self, spam=False):
         data = {"id": self.thing.fullname, "spam": bool(spam)}
         self.thing._reddit.post(API_PATH["remove"], data=data)
 
+    def removal_reasons(self):
+        """Return list of available removal reasons.
+
+        reason ids are required in order to use :meth:`.add_removal_reason`.
+
+        For example:
+
+        .. code:: python
+
+           removal_reasons = submission.mod.removal_reasons()
```

Comment:
That was my mistake. I forget to replace self.submission with self.thing since these should work for comments and submissions (hence why they are in ThingModerationMixin